services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/entity/MulticastGoalStateV2.java

```diff
@@ -16,19 +16,16 @@
 
 package com.futurewei.alcor.dataplane.entity;
 
+import com.futurewei.alcor.dataplane.exception.HostIdNotFound;
+import com.futurewei.alcor.dataplane.exception.VpcIdNotFound;
 import com.futurewei.alcor.schema.Goalstate.GoalStateV2;
 import com.futurewei.alcor.web.entity.dataplane.MulticastGoalStateByte;
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
 
 public class MulticastGoalStateV2 {
     private Set<String> hostIps;
```

Comment:
In my opinion, Set of hostIps can be removed. The getter of hostIps can be implemented by `hostVpcMap.keySet()`

Reply:
@VanderChen Let's make a comment here and we can make change in the next release.

Reply:
In MulticastGoalStateV2, a Set holds the hostIps needed by gRPC path and a HashMap holds the (HostIp, VpcId) pairs for Pulsar path. In my opinion, the Set of hostIps can be removed, and HashMap.keySet() can be used to get hostIps for gRPC path.

Reply:
@VanderChen  I see. If we remove this line of code now, you probably also need to modify lots of code in many different places. If this is the case, we better make a comment here and change it later in another PR.